libpod/container.go

```diff
@@ -718,6 +728,21 @@ func (c *Container) Export(path string) error {
 	return err
 }
 
+// AddArtifacts creates and writes to an artifact file for the container
+func (c *Container) AddArtifacts(name string, data []byte) error {
+	return ioutil.WriteFile(filepath.Join(c.config.StaticDir, "artifacts", name), data, 0740)
```

Comment:
Can you make a container.getArtifactPath(name string) function so we can share the filepath.Join between all three of these?

Reply:
fixed